csharp/Platform.IO/ConsoleHelpers.cs

```diff
@@ -38,9 +82,29 @@ public static string GetOrReadArgument(int index, string readMessage, params str
             }
         }
 
+        /// <summary>
+        /// <para>Outputs a <paramref name="string" /> to the console. The method is only executed if the application was compiled with the DEBUG directive.</para>
+        /// <para>Выводит строку(<paramref name="string" />) в консоль. Метод выполняется только в том случае, если приложение было скомпилировано с директивой DEBUG.</para>
+        /// </summary>
+        /// <param name="string">
+        /// <para>The string to be outputted to the console.</para>
```

Comment:
`The string to output to the console.`, this version [can be translated back](https://translate.google.com/?sl=auto&tl=ru&text=The%20string%20to%20output%20to%20the%20console.&op=translate) and is shorter, therefore it is better.